tests/model_serving/model_server/utils.py

```diff
@@ -217,3 +225,100 @@ def run_inference_multiple_times(
 
             if exceptions:
                 raise InferenceResponseError(f"Failed to run inference. Error: {exceptions}")
+
+
+def verify_keda_scaledobject(
+    client: DynamicClient,
+    isvc: InferenceService,
+    expected_trigger_type: str | None = None,
+    expected_query: str | None = None,
+    expected_threshold: str | None = None,
+) -> None:
+    """
+    Verify the KEDA ScaledObject.
+
+    Args:
+        client: DynamicClient instance
+        isvc: InferenceService instance
+        expected_trigger_type: Expected trigger type
+        expected_query: Expected query string
+        expected_threshold: Expected threshold as string (e.g. "50.000000")
+    """
+    scaled_objects = get_isvc_keda_scaledobject(client=client, isvc=isvc)
+    scaled_object = scaled_objects[0]  # Get the first ScaledObject
+    trigger_meta = scaled_object.spec.triggers[0].metadata
+    trigger_type = scaled_object.spec.triggers[0].type
+    query = trigger_meta.get("query")
+    threshold = trigger_meta.get("threshold")
+
+    assert trigger_type == expected_trigger_type, (
+        f"Trigger type {trigger_type} does not match expected {expected_trigger_type}"
+    )
+    assert query == expected_query, f"Query {query} does not match expected {expected_query}"
+    assert threshold == expected_threshold, f"Threshold {threshold} does not match expected {expected_threshold}"
+
+
+def run_vllm_concurrent_load(
+    pod_name: str,
+    isvc: InferenceService,
+    response_snapshot: Any,
+    chat_query: list[list[dict[str, str]]],
+    completion_query: list[dict[str, str]],
+    num_concurrent: int = 5,
+    duration: int = 120,
```

Comment:
The `pod_name`, `response_snapshot`, `chat_query` and `completion_query` arguments are unused.

So, I think this `run_vllm_concurrent_load` func can be merged with `run_ovms_concurrent_load` by removing these unused args, and adding as args the differences for `inference_config` and `inference_type`.